applications/nrf5340_audio/src/bluetooth/le_audio_cis_gateway.c

```diff
@@ -207,13 +228,16 @@ static void stream_configured_cb(struct bt_audio_stream *stream,
 	int ret;
 	uint8_t channel_index;
 
-	ret = stream_index_get(stream, &channel_index);
+	ret = headset_conn_index_get(stream->conn, &channel_index);
 	if (ret) {
 		LOG_ERR("Stream not found");
-	} else {
+		return;
+	}
+
+	if (unicast_group) {
```

Comment:
One more thing: What is the purpose of this if?

Reply:
Since bt_audio_stream_qos takes in unicast_group as a variable we need the group to be created before we set the qos. stream_configured_cb can be called multiple times, ones for sink and then once again for source.